scale/job/test/test_views.py

```diff
@@ -3921,6 +3734,7 @@ def test_bad_custom_resources(self):
         response = self.client.generic('POST', url, json.dumps(json_data), 'application/json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, response.content)
 
+    # trigger rules are ignored in Scale v6, so no need to check them
```

Comment:
can probably remove this comment as it's confusing without trigger rules anywhere